.github/workflows/tests.yml

```diff
@@ -571,6 +571,7 @@ jobs:
         env:
           POSTGRES: ${{ (matrix.database == 'Postgres') && 1 || '' }}
           WORKERS: ${{ (matrix.arrangement == 'workers') && 1 || '' }}
+          UNIX_SOCKETS: 1
```

Comment:
Should we worry about testing both HTTP and unix sockets?

Reply:
Short term: no
Long term: perhaps, if the nightly experimental run of various testing ever gets implemented, that would be a prime spot to put something like this. The resource of Github CI runners is finite, and there is most likely no point in testing this for every PR and such.

(This is actually just for testing while discussing this PR, the commit that places this here is to be reverted before merge. I think I forgot(again) to mention that this pull request could be reviewed per commit.) See [e14422f](https://github.com/matrix-org/synapse/pull/15708/commits/e14422fe044f95972d428bff4bbe9e3aba03c96f)

Reply:
Reverted in [0ae6036](https://github.com/matrix-org/synapse/pull/15708/commits/0ae603683fc135395fc882579b3da95074e84bcd)

That should finish this up.